server/routes/partyRoutes.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import PartyController from '../controllers/partyController';
+import PartyMiddleware from '../middlewares/partyMiddleware';
+
+const router = express.Router();
+
+const apiUrl = '/api/v1/parties';
+
```

Comment:
@frankly034 Nice work can please `version` your API?

Reply:
I've versioned the api. Love to have your review on it. Thanks